# HG changeset patch
# User Brian <runningnak3d@gmail.com>
# Date 1277924723 21600
# Branch trunk
# Node ID a236b377aa81101e68a82be1cc11d1b9764e5cf1
# Parent  e93c431961731f1bf191e6dd2e0fff52a8a61405
* Another opcode update
* Thanks TOM_RUS

diff -r e93c43196173 -r a236b377aa81 src/server/game/Server/Protocol/Opcodes.h
--- a/src/server/game/Server/Protocol/Opcodes.h	Wed Jun 30 13:02:43 2010 -0600
+++ b/src/server/game/Server/Protocol/Opcodes.h	Wed Jun 30 13:05:23 2010 -0600
@@ -1244,7 +1244,7 @@
     SMSG_ITEM_REFUND_RESULT                         = 0x4B5, // refund item result
     CMSG_CORPSE_MAP_POSITION_QUERY                  = 0x4B6, // CMSG, uint32
     SMSG_CORPSE_MAP_POSITION_QUERY_RESPONSE         = 0x4B7, // SMSG, 3*float+float
-    CMSG_LFG_SET_ROLES_2                            = 0x4B8, // not found
+    UMSG_UNKNOWN_1208                               = 0x4B8, // not found
     UMSG_UNKNOWN_1209                               = 0x4B9, // not found
     CMSG_CALENDAR_CONTEXT_EVENT_SIGNUP              = 0x4BA, // CMSG, uint64, lua: CalendarContextEventSignUp
     SMSG_CALENDAR_ACTION_PENDING                    = 0x4BB, // SMSG, calendar related EVENT_CALENDAR_ACTION_PENDING
